### ISSUE TITLE
Issue with  IsomirDataSeqFromFiles

### ISSUE THREAD

**Issue by biowizz** [lpantano/seqcluster#23]
Hello,

i am trying to use the package on smallRNA seq data. I have used miraligner and got *.mirna output.
Now I want to use the isoPlot( ) on our data. I tried with the example files given and used the commands as given below it works fine.
```
path <- system.file("extra", package="isomiRs")
fn_list <- list.files(path, full.names = TRUE)
de <- data.frame(row.names=c("f1" , "f2"), condition = c("newborn", "newborn"))
ids <- IsomirDataSeqFromFiles(fn_list, design=de)
```

Now, I am trying the same for our data and it is showing the error as follows.

```
Error in FUN(X[[i]], ...) : 
  assay colnames() must be NULL or equal colData rownames()
In addition: Warning message:
In IsoCountsFromMatrix(listSamples, design) : No miRNA found.
```
Could you please suggest any changes I need to make. One observation, The frequency column in our data has only 0. Rest all the column are same as the example data. I also deleted the one addtional precursor column as it was not present in example data.

**Comment by lpantano** [comment]
Hi,

Sorry for this issue.

I think the problem is the column for frequency to be 0.

How did you analyze your data from the beginning? Normally, there is a collapsing step where you end up with a fasta/fastq file with this format:

>seq_ID_x100
SEQUENCE

where x100 is the count data. This step is mandatory if you want to quantify miRNAs and use isomiRs. If you tell me each step you did from the raw FASTQ files I can help you to fix the problem.

Thanks!

**Comment by biowizz** [comment]
Hi Ipantano,
Thank you for the quick assistance. The reads that I have are single end from ion Torrent. I have used the manufacturer recommended methodology to do the QC on our data and have the fastq files from them. I used these fastq files in java based miraligner to obtain *.mirna files.
I have not used the seqcluster for the entire analysis rather only miraligner on fastq files. Is that a problem? 
Which steps should I follow to obtain the fastq files ready for miraligner?

Thanks
